fml/backtrace_unittests.cc

```diff
@@ -4,34 +4,20 @@
 
 #include "backtrace.h"
 
+#include <csignal>
+
 #include "gtest/gtest.h"
-#include "logging.h"
 
 namespace fml {
 namespace testing {
 
-TEST(BacktraceTest, CanGatherBacktrace) {
+TEST(BacktraceDeathTest, CanGatherBacktrace) {
```

Comment:
This test does nothing useful now. Let's remove this as well as `IsCrashHandlingSupported`.

Reply:
done